assets/build/install.sh

```diff
@@ -113,7 +117,9 @@ if [[ -d ${GEM_CACHE_DIR} ]]; then
   mv ${GEM_CACHE_DIR} ${GITLAB_INSTALL_DIR}/vendor/cache
   chown -R ${GITLAB_USER}: ${GITLAB_INSTALL_DIR}/vendor/cache
 fi
-exec_as_git bundle install -j$(nproc) --deployment --without development test aws
+
+exec_as_git bundle lock --update=omniauth-google-oauth2
```

Comment:
This should be a separate commit. To make an easier revert if it's working correctly in the upstream again.